Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -14595,6 +14598,172 @@ def pan_os_get_master_key_details_command() -> CommandResults:
     )
 
 
+def pan_os_get_certificate_info_command(topology: Topology, args: Dict) -> CommandResults:
```

Comment:
This function is about 160 line, except of moving the nested 2 function to the outer scope
consider dividing this main function ("pan_os_get_certificate_info_command") to a few sub-functions which every sub function has its job, then this main function should call them in order.